python/py-curio/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-curio
+version             1.5
+revision            0
+license             BSD
+maintainers         nomaintainer
+
+description         Curio is a coroutine-based library for concurrent systems programming
+long_description    {*}${description}. It provides standard programming \
+                    abstractions such as as tasks, sockets, files, locks, \
+                    and queues as well as some advanced features such as \
+                    support for structured concurrency.
+
+homepage            https://github.com/dabeaz/curio
+
+checksums           rmd160  4f422c83ccfa3b13a771eb745bb3f5d61fa67201 \
+                    sha256  af08212e590bb7da8e4cc39c42012711494dc20d622f162155ba296cc2e3bc10 \
+                    size    234920
+
+python.versions     38 39 310
+
+if {${name} ne ${subport}} {
+    depends_run-append  \
+                    port:py${python.version}-setuptools
```

Comment:
why is this a runtime dependency? I see it listed in `setup.py` in a `try` clause, it appears you decided not to use it as a build dependency; for what is it used at runtime?

Reply:
Oops. Fixed thanks.